src/lib/lumi.ts

```diff
@@ -2022,6 +2022,22 @@ export const fromZigbee = {
             }
         },
     } satisfies Fz.Converter,
+    lumi_pressure: {
+        cluster: 'msPressureMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            let pressure = 0;
+            if (msg.data.hasOwnProperty('scaledValue')) {
```

Comment:
I think this part can to into the official `fz.pressure`

Then call it from here, example:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/8dce09a4caa4258040384f68bbdcb77a22b0a55e/src/devices/tuya.ts#L352

Reply:
Yes, true enough.